.github/workflows/release_droid_prepare_original_checksum.yml

```diff
@@ -32,7 +32,7 @@ jobs:
             ${{ runner.os }}-go-${{ matrix.go }}
             ${{ runner.os }}-go-
       - name: Install Go tools
-        run: go install github.com/google/go-licenses@latest
+        run: go install github.com/google/go-licenses@v1.2.1
```

Comment:
Would it make sense to define the actual version in a single place a use only references in the *.yml files to this version?

Reply:
yes, we have an issue for installing it automaticlly.

Reply:
https://github.com/exasol/project-keeper/issues/323